<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Currency Exchange</string>
        </property>
        <widget class="QWidget" name="centralwidget"/>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>22</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuFile">
                <property name="title">
                    <string>&amp;File</string>
                </property>
                <addaction name="actionExit"/>
            </widget>
            <widget class="QMenu" name="menuActions">
                <property name="title">
                    <string>&amp;Actions</string>
                </property>
                <addaction name="actionExchange"/>
                <addaction name="actionRequests"/>
                <addaction name="actionProfile"/>
            </widget>
            <addaction name="menuFile"/>
            <addaction name="menuActions"/>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
        <action name="actionExit">
            <property name="text">
                <string>&amp;Exit</string>
            </property>
        </action>
        <action name="actionExchange">
            <property name="text">
                <string>&amp;Exchange</string>
            </property>
        </action>
        <action name="actionRequests">
            <property name="text">
                <string>&amp;My Requests</string>
            </property>
        </action>
        <action name="actionProfile">
            <property name="text">
                <string>&amp;Profile</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>
